upup/pkg/fi/cloudup/awstasks/vpc.go

```diff
@@ -44,9 +44,14 @@ type VPC struct {
 	Shared *bool
 
 	Tags map[string]string
+
+	// RemoveExtraCIDRBlocks contains a list of cidr blocks that should be
+	// associated with the VPC; any other CIDR blocks should be disassociated.
+	RemoveExtraCIDRBlocks []string
```

Comment:
To me, the name of the variable suggests it contains which CIDRs to delete. Maybe `AssociateExtraCIDRBlocks` or `KeepCIDRBlocks` are better names?

Reply:
Agreed the variable name is not ideal, I just updated to `AssociateExtraCIDRBlocks`. For reference I was following how SecurityGroup handles it:
https://github.com/kubernetes/kops/blob/b9921511a46a4b270073e18a676520cbd819f243/upup/pkg/fi/cloudup/awstasks/securitygroup.go#L42

but we should probably update that variable as well.